front/app/containers/EventsShowPage/components/EventSharingButtons/messages.ts

```diff
@@ -6,7 +6,7 @@ export default defineMessages({
     defaultMessage: 'Attend this event: {eventTitle}',
   },
   emailSharingBody: {
-    id: 'app.containers.EventsShow.emailSharingBody',
+    id: 'app.containers.EventsShow.emailSharingBody2',
```

Comment:
Noticed that the text for this had changed last year, but the translations had not been updated